doc/developers/maintainer.rst

```diff
@@ -66,26 +66,31 @@ under that branch.
 
 In terms of including changes, the first RC ideally counts as a *feature
 freeze*. Each coming release candidate and the final release afterwards will
-include minor documentation changes and bug fixes. Any major enhancement or
-feature should be excluded.
+include only minor documentation changes and bug fixes. Any major enhancement
+or feature should be excluded.
 
 The minor releases should include bug fixes and some relevant documentation
 changes only. Any PR resulting in a behavior change which is not a bug fix
 should be excluded.
 
-First, create a branch, **on your own fork** (to release e.g. `0.999.3`):
+For major releases, do not forget to prepare a Release Highlights page as a
+runnable example and check that its HTML rendering looks correct. These release
+higlights should be linked from the ``whats_new.rst`` file for the new version
+of scikit-learn.
```

Comment:
```suggestion
higlights should be linked from the ``doc/whats_new/v0.99.rst`` file for the new version
of scikit-learn.
```

Reply:
The [what's new page](https://scikit-learn.org/stable/whats_new.html) does not contain the link I think..